services/QuillLMS/app/controllers/blog_posts_controller.rb

```diff
@@ -102,6 +106,7 @@ def show_topic
     topic = CGI::unescape(params[:topic]).gsub('-', ' ').capitalize
     return if BlogPost::TOPICS.include?(topic)
     return if BlogPost::STUDENT_TOPICS.include?(topic)
+    return if params[:topic] == BlogPost::WHATS_NEW_SLUG
```

Comment:
I wonder if we want a specific constant of `INVALID_TOPICS` or something.  That's not necessarily better, but does make it clear that all of our `return if` conditions are really checking for the same idea.  You could set:
```ruby
  private def redirect_invalid_topics
    topic = CGI::unescape(params[:topic]).gsub('-', ' ').capitalize
    valid_topics = BlogPost::TOPICS + BlogPost::STUDENT_TOPICS + [BlogPost::WHATS_NEW_SLUG]
    return if valid_topics.include?(topic)

    ...
end
```